<launch>

    <node name="map_server" pkg="map_server" type="map_server" args="/home/stepan/Projects/ORCAStar/ORCAStar/examples/text.yml" />

    <node pkg="ORCAStar" name="ROSMission" type="ROSMission">
        <param name="agents_number" type="int" value="5"/>
    </node>

    <node name="$(anon ROSSimActor_0)" pkg="ORCAStar" type="ROSSimActor"/>
    <node name="$(anon ROSAgent_0)" pkg="ORCAStar" type="ROSAgent">
        <param name="param" type="int" value="0"/>
    </node>

    <node name="$(anon ROSSimActor_1)" pkg="ORCAStar" type="ROSSimActor"/>
    <node name="$(anon ROSAgent_1)" pkg="ORCAStar" type="ROSAgent">
        <param name="param" type="int" value="1"/>
    </node>

    <node name="$(anon ROSSimActor_2)" pkg="ORCAStar" type="ROSSimActor"/>
    <node name="$(anon ROSAgent_2)" pkg="ORCAStar" type="ROSAgent">
        <param name="param" type="int" value="2"/>
    </node>

    <node name="$(anon ROSSimActor_3)" pkg="ORCAStar" type="ROSSimActor"/>
    <node name="$(anon ROSAgent_3)" pkg="ORCAStar" type="ROSAgent">
        <param name="param" type="int" value="3"/>
    </node>

    <node name="$(anon ROSSimActor_4)" pkg="ORCAStar" type="ROSSimActor"/>
    <node name="$(anon ROSAgent_4)" pkg="ORCAStar" type="ROSAgent">
        <param name="param" type="int" value="4"/>
    </node>



</launch>
